src/Components/Authentication/Footer.tsx

```diff
@@ -13,14 +13,24 @@ const FooterContainer = styled.div`
 `
 
 export const Footer = props => {
-  const { onFacebookLogin, handleTypeChange, mode, inline } = props
+  const {
+    onFacebookLogin,
+    onTwitterLogin,
+    handleTypeChange,
+    mode,
+    inline,
+  } = props
   switch (mode) {
     case "login": {
       return (
         <FooterContainer inline={inline}>
-          <SmallTextLink onClick={onFacebookLogin}>
-            Log in using Facebook.
-          </SmallTextLink>
+          <SmallText>
+            {"Log in using "}
+            <SmallTextLink onClick={onFacebookLogin}>Facebook</SmallTextLink>
+            {" or "}
+            <SmallTextLink onClick={onTwitterLogin}>Twitter</SmallTextLink>
+          </SmallText>
+
```

Comment:
Possible to add "." at the end of this sentence? https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROW&modal=detail&selectedIssue=GROW-726

Reply:
Yup